### DISCUSSION TITLE
DirecTV power on

### DISCUSSION THREAD

**Discussion by tryin100** [GerryDazoo/Slinger#360] [Q&A]
All is working except the 'power' button does not turn on the DirecTV box (it does power on the slingbox 500).
I am using the custom remote remote_directv-h24.txt.
Any advice appreciated.

**Comment by sp222222** [comment]
> All is working except the 'power' button does not turn on the DirecTV box (it does power on the slingbox 500). I am using the custom remote remote_directv-h24.txt. Any advice appreciated.

I could send you my dtv_H24 txt (I made myself, nothing fancy) but unless your input on the 500 is configured and saved correctly to the slingbox500 itself, it's all a moot point using any txt/remote file. 

I've not used this file but I'm guessing you got it from this site someone posted not too long ago?

**Comment by tmcafee52** [comment]
What DirecTV box are you trying to use the power button on?  I just tested the remote_directv-h24.txt and remote_directv-h24_large_digits.txt files with the H24 and they both work fine.  I do have to restart the stream in VLC when I power it back on.

**Reply by tryin100** [reply]
I'd be happy to try your dtv_h24 file, thanks.

I have two DirecTV 'genie' (small) boxes models c51700 and c51500.  When I start the VLC stream they DO NOT power on automatically.
Both those boxes use the small (older) DirecTV remote.
For these I am now able to use the slinger remote to power on the DTV box.
I believe I am now able to do this because I added the line in config.ini, under [remote], "code=1"
 
I have a DirecTV (large) box model HR24-200.  When I start the VLC stream it powers on the DTV box automatically.
This box uses the larger (newer) DirecTV remote.

I have found, however, that slinger does not seem to be loading the directv 'custom' remote any longer.
The 'remote_directv-h24.txt' file that I use is actually from this site (it along with others can be found in the 'custom remote' folder when you unzip the download).

in the old config.sys the command was "remote=remote_directv-h24.txt", 
in the newer config.sys the command to include is now  "include=remote_directv-h24.txt"
Although whatever remote is loading seems to work it is definitely different now.  I can tell because the buttons are different.

**Comment by tmcafee52** [comment]
I tried the unified configuration file with the remote_directv-h24_large_digits.txt file and the power button did not work.  It does work with the original config file.  The original file has a section for the [REMOTE] section that contains a statement for the remote code.  For my ProHD, the statement is "Code=3".   I added a statement in the unified file in each of the slingbox sections to specify the RemoteCode and the power button now works.  For my ProHD, I added "RemoteCode=3" in my slingbox section [ProHD].

**Reply by GerryDazoo** [reply]
The remote code "should" be the same as your video source. If it's not it means you've configured multiple video sources on your box and you have a mismatch between the videosource you've selected in your config.ini and what you actually have on your box. 
You "did" configure a VideoSource in your file? This is the only reason you would need to override the currently selected videosource with the  a different remote code.

**Reply by tmcafee52** [reply]
As you expected, I didn't define a videosource in the ProHD section.  The videosource statement was commented out.  It works fine now with the videosource statement defined and with the RemoteCode statement removed.  Thanks again for all your help.
